api/wasm/v0/cpp/proxy_wasm_externs.h

```diff
@@ -0,0 +1,117 @@
+/*
+ * Proxy-WASM ABI.
+ */
+// NOLINT(namespace-envoy)
+
+#pragma once
+
+#include <stddef.h>
+#include <stdint.h>
+
+//
+//  ABI calls from the host into the VM.
```

Comment:
Can you add some comments on who is supposed to implement these? You can also leave a forward reference to an example for when it lands in later PRs.

Reply:
Done.